x-pack/README.md

```diff
@@ -71,6 +69,14 @@ To do all of this in a single command run:
 node scripts/functional_tests
 ```
 
+#### Plugin functional and api integration tests (i.e. "x-pack" directory tests)'
+
+To quickly run a single functional or api-integration test or group of tests from the x-pack directory, try the following:
+```sh
+cd x-pack/ && node scripts/functional_tests_server.js
+node scripts/functional_test_runner.js --config x-pack/test/{api_integration,functional}/config --grep "my fancy test"
+```
+
```

Comment:
This is described in the [here](../scripts/README.md) the next section "Developing functional UI tests" links to. Not sure if any author making changes to that would also think to update this. What does the duplication add?

Reply:
That link is a 404, can you help me understand where this is duplicated?

Reply:
As for the "Developing functional UI tests" section -- I didn't want to change this page too drastically, but I think it's less about "developing tests" and more about running individual or smaller groups of tests (I can see someone wanting to do that in many cases that don't have to do with specifically developing new tests, etc). I don't see anything about how to easily do that in this page. The lines you sent me are so clear and useful (in contrast with how deep everything else goes on this page), it seems weird to leave them out of the docs.

Reply:
I don't know how I messed up that URL :see_no_evil: I meant to point to https://github.com/elastic/kibana/blob/master/scripts/README.md.